_automation/main.go

```diff
@@ -389,6 +432,165 @@ func (s *UpdateService) downloadTypescript(ctx context.Context, g *Grammar) {
 	}
 }
 
+// Add this function to your UpdateService methods
```

Comment:
this is a strange comment

Reply:
💯 This code needs some cleanup, clearly.

Reply:
💯 This code needs some cleanup, clearly.